src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/197403779-6241c220-1105-44ce-a9e6-bdc3add0ae9d.png)

Reply:
The reduntant empty lines were deleted from  my code.